homeassistant/components/fjaraskupan/config_flow.py

```diff
@@ -0,0 +1,47 @@
+"""Config flow for Fjäråskupan integration."""
+from __future__ import annotations
+
+import asyncio
+from typing import Any
+
+from bleak import BleakScanner
+from bleak.backends.device import BLEDevice
+from bleak.backends.scanner import AdvertisementData
+from fjaraskupan import device_filter
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+CONST_WAIT_TIME = 5.0
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Fjäråskupan."""
+
+    VERSION = 1
+
+    def __init__(self):
+        """Initialize conflig flow."""
+        self._devices: dict[str, str] = []
+
+    async def async_step_user(
```

Comment:
Can we use the discovery flow helper here instead?

https://github.com/home-assistant/core/blob/32a2c5d5dbdeacbeeb58e62458bbb40bee07d945/homeassistant/helpers/config_entry_flow.py#L104-L110

Reply:
Indeed! Neat!